andela/titan-dev-ah-frontend
# #163518663 Report Article

#### What does this PR do?
- Report article
#### Description of Task to be completed?
- Enable report article
#### How should this be manually tested?
- clone the repos and check out the branch `ft-report-article-163518663
- Run `yarn` to install all dependencies
- Run `yarn test` to run tests, they should all pass
- Start your backend server on the address `localhost:3000/`
- Create some articles using client tool like the postman 
- Grab slug of one of the articles you created. 
- Run `yarn start` to start your server
- Navigate to `localhost:8080/articles/slug`
- Report the article, you should see the response explaining whether the article was reported successfully or not. 
#### Any background context you want to provide?
- The html-css mockup of this story was not yet implemented. So, I designed the mockup and I will include it. 
#### What are the relevant pivotal tracker stories?
# 163518663 
#### Screenshots (if appropriate)
<img width="1368" alt="Screen Shot 2019-05-02 at 16 20 19" src="https://user-images.githubusercontent.com/34691070/57470091-58a77900-7288-11e9-8a85-532fbfe7bab9.png">
